index.html

```diff
@@ -169,5 +169,50 @@ <h4 class="profile-tittle">Skills</h4>
             </article>
         </div>
     </section>
+
+    <section class="form-section">
+        <div class="header-img"></div>
+        <div class="form-container container">
+            <h1 class="tittle">Get started</h1>
+            <p class="para para2">
+                Lorem ipsum dolor sit amet, consectetur adipiscing
+                elit, sed do eiusmod tempor incididunt ut labore
+                et dolore magna.
+            </p>
+            <form class="form-input" action="https://formspree.io/f/mpzndezj" method="post">
+                <div class="float-container">
+                    <label for="name">Your name</label>
+                    <input type="text" id="name" name="name" maxlength="30" required>
+                </div>
+                <div class="float-container">
+                    <label for="email">Your email</label>
+                    <input type="email" id="email" name="email" required
+                    pattern="[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,6}$">
+                </div>
+                <div class="float-container">
+                    <label for="message">Enter your message here</label>
+                    <textarea name="message" id="message" placeholder="Enter your message here" cols="30" rows="10" maxlength="500" required>                   </textarea>
+                </div>
+                <button type="submit" class="cta">Start collaboration</button>
+            </form>
+        </div>
+    </section>
+
+    <footer class="footer-section">
+        <div class="footer-container">
+            <div class="get-resume">
+                <a class="footer-tittle" href="#">Get my Resume</a>
+                <a href="#"><img class="resume-img" src="./images/ic-link-teal.svg" alt="ink teal"></a>
+            </div>
+            <div class="social-media">
+                <a href="#"><img class="social-link" src="./images/ic-stackoverflow.svg" alt=""></a>
+                <a href="#"><img class="social-link" src="./images/ic-github.svg" alt=""></a>
+                <a href="#"><img class="social-link" src="./images/ic-twitter.svg" alt=""></a>
+                <a href="#"><img class="social-link" src="./images/ic-linkedin.svg" alt=""></a>
+                <a href="#"><img class="social-link" src="./images/ic-medium.svg" alt=""></a>
+                <a href="#"><img class="social-link" src="./images/ic-angellist.png" alt=""></a>
+            </div>
```

Comment:
- For semantic purposes and accessibility, I'd suggest you wrap your links in a `ul` tag with each link in a `li` tag. Please try to fix this 🙏.